language/bytecode_verifier/bytecode_verifier_tests/src/unit_tests/signature_tests.rs

```diff
@@ -92,3 +94,166 @@ proptest! {
         prop_assert_eq!(expected_violations, actual_violations);
     }
 }
+
+#[test]
+fn no_verify_locals_good() {
+    let compiled_module_good = CompiledModuleMut {
```

Comment:
I think using @vgao1996's dummy module [code](https://github.com/libra/libra/blob/master/language/vm/src/file_format.rs#L1719) might save you some boilerplate here.

Reply:
I looked at the dummy module code and decided not to use it.  There are a bunch of cross-linkages to keep track of in these tests (thanks @tzakian).  I found it easier to see the module displayed in front of me to make sense of the test.